.github/workflows/R-CMD-check.yaml

```diff
@@ -7,9 +7,11 @@
 on:
   push:
     branches:
+      - main
       - master
   pull_request:
     branches:
+      - main
       - master
```

Comment:
* We are now specific about what branches trigger gh-actions. This saves runners and time when,  
  for example, we push a branch but don't yet care about running CI. If we do want to trigger CI we
  can now more intentionally make it a PR -- maybe as a draft.

* GitHub is pushing to rename the default branch from master to main. As we transition, we need
  to watch both.